metron-interface/metron-config/scripts/package.json

```diff
@@ -9,7 +9,8 @@
     "http-proxy-middleware": "0.17.4",
     "optimist": "0.6.1",
     "serve-favicon": "2.4.2",
```

Comment:
( I found a command on stack overflow : https://stackoverflow.com/questions/16274841/output-all-licenses-of-installed-node-js-libraries

find * -name package.json | xargs grep '"license[s]*"' -A 3

--------------------------------------------------
[node_dependencies.txt](https://github.com/apache/metron/files/1030623/node_dependencies.txt)


The lic are : 

- MIT, Apache 2.0
-  bsd-3-clause
- ( bsd-3-clause OR MIT)
- (BSD-3-Clause AND MIT)
-  ISC, BSD
- bsd-4-clause
- MIT/X11
- bsd-2-clause
- AFLv2.1
- (BSD-2-Clause OR MIT OR Apache-2.0)
- (MIT AND CC-BY-3.0)
- Unlicense
- "SEE LICENSE IN COPYING.txt",
- CC-BY-3.0
- CC-BY-4.0
- BSD-like
- (OFL-1.1 AND MIT)
- (BSD-2-Clause OR MIT OR Apache-2.0)
- (WTFPL OR ISC)
- (WTFPL OR MIT)
- WTFPL
- Public Domain

I am not familiar with all of these and their compatibility with apache